deco-sites/otica-isabela
# fixed product details thumbs adding arrows to change to the next image

## What is this contribution about?



## How to test it?

Go to any PDP and check the thumbs, now there is arrow to scroll horizontally the images.
